src/js/components/sharedComponents/Footer.jsx

```diff
@@ -71,6 +71,11 @@ export default class Footer extends React.Component {
                                             link="http://fedspendingtransparency.github.io/DAIMS-v1.1/"
                                             title="Data Model" />
                                     </li>
+                                    <li>
+                                        <FooterExternalLink
+                                            link="https://fedspendingtransparency.github.io/data-lab/"
```

Comment:
I'm seeing a 404 when I follow this link

Reply:
Checked earlier today that this is the correct URL, supposedly this site will be ready when we deploy. If it's not ready by then, I'll remove.

Reply:
Not deployed yet